### ISSUE TITLE
Files and folders are not hidden until I recharge the plugin manually

### ISSUE THREAD

**Issue by Mara-Li** [Oliver-Akins/file-hider#12]
### Affected Platforms

iOS, Windows

### Expected Behaviour

Selected file are hidden 

### Actual Behaviour

File are not hidden until I recharge manually the plugin or re-hide the files/folder. 

### Have you?

- [X] I have checked that there isn't already a bug on GitHub for this issue

### Urgency

Mildly Urgent

**Comment by Oliver-Akins** [comment]
I'm not sure I understand what you mean by "recharge the plugin", is this the toggle switch in the "Community Plugins" tab of the Obsidian settings?

**Comment by Mara-Li** [comment]
Hello !
I use the 1.1.0 version, and you're right : i need to manually unload / load the plugin through obsidian community plugin.

**Comment by Oliver-Akins** [comment]
Okay, thanks for the info; I'll take a look and see what I can do about it.

**Comment by SuperTempAccount** [comment]
Hey, I'm experiencing the same issue. A few more details that might help narrow down the issue:

1. The issue occurs the next day from what I can see (no need to restart Obsidian)
2. Toggling the hide option is enough to fix the issue for the day (no need to unload/load the plugin) 
![image](https://user-images.githubusercontent.com/83566267/188332336-57b4c209-402b-4c63-ba6b-246c8ab8ff34.png)
3. The files I'm hiding are on OneDrive

Thanks for making this plugin. It keeps my workplace clean and enjoyable.
Best,

**Comment by Lisandra-dev** [comment]
Yeah, I continue to have this problem... I prefer to use CSS, it takes one line in my snippet css to hide. At last it keeps to work.

**Comment by Oliver-Akins** [comment]
Yeah, the way the plugin works is odd, I have been attempting to make it more reliable and am aware of the issues that still happen (at least those that have been mentioned in GH issues), but life's been busy and whatnot so the times I get to work on it aren't super frequent. 

(Though I have not forgotten / abandoned this plugin, so it will be updated to fix these issues, I just can't give a solid timeline for said fixes)

**Comment by callumhackett** [comment]
Just adding that I see this issue also (a file in the root folder is not hidden until the plugin's file visibility status is manually toggled on every restart). Perhaps you could update the readme so it's clear to users that the automated features of the plugin are unavailable? That is the main reason to use it.

**Comment by Oliver-Akins** [comment]
@callumhackett what version of the plugin are you using? If it's 1.1.0, try updating to 1.1.1 and if the issue still happens let me know.

**Comment by callumhackett** [comment]
> @callumhackett what version of the plugin are you using? If it's 1.1.0, try updating to 1.1.1 and if the issue still happens let me know.

This is version 1.1.1. I did some extra testing and it appears that the issue is persistent on Android but not on Windows. The plugin works manually on Android by toggling the file visibility status but files will appear unhidden on app restart. On Windows, the behaviour is as expected.
